codes/classical/q-ary_digits/justesen.yml

```diff
@@ -0,0 +1,118 @@
+#######################################################
+## This is a code entry in the error correction zoo. ##
+##       https://github.com/errorcorrectionzoo       ##
+#######################################################
+
+# Use UTF-8 unicode encoding
+# AMS-TeX commands are rendered inside \( ... \) using MathJaX.
+# Allowed external bibliographic references are
+#    \cite{arXiv:#.#} or \cite{arXiv:quant-ph/#} (PREFERRED),
+#    \cite{doi:#}, or, as a last resort
+#    \cite{manual:{(enter citation line incl. author and year here)}}
+# External websites such as code tables, coding theory packages, github pages linked as
+#    \url{https://example.com/example}
+#    \href{https://example.com/example}{link text}
+# Internal references to codes are
+#    \hyperref[code:code_id]{link text}
+# Delete instructional comments when submitting
+
+# code id, physical, logical are all lower case
+code_id: justesen
+physical: q-ary_digits
+logical: q-ary_digits
+
+# Apostrophes are denoted by two apostrophe characters, i.e., ''
+# Code title (SINGULAR) + first reference(s) (optional).
+name: 'Justesen_code'
+introduced: '\cite{manual:{MacWilliams/Sloane, The Theory of Error-Correcting Codes, Sec.10.11}}'
```

Comment:
The correct reference is J. Justesen's paper of about 1972 on "A class of constructive asymptotically good codes" (I am citing the title from memory). They are indeed presented in the cited book, among other places.